docs/hascontext.md

```diff
@@ -1,16 +1,17 @@
 ---
-id: hascontext
-title: HasContext
+id: withcontext
```

Comment:
Filename must be the same as id, otherwise docusaurus won't show the page.

Reply:
heck

Reply:
fixed